src/Parsing/Impl/Tokenizer.cs

```diff
@@ -2302,9 +2287,10 @@ struct State : IEquatable<State> {
                     NextWhiteSpace = null;
                 }
                 GroupingRecovery = null;
+                IndentFormat = new string[maxIndent];
             }
 
-            public State(TokenizerOptions options) {
+            public State(TokenizerOptions options, int maxIndent) {
                 Indent = new int[MaxIndent]; // TODO
```

Comment:
Is `maxIndent` needed if `MaxIndent` exists? I thought `MaxIndent` was used previously to initialize `IndentFormat`, just in the wrong place.

Reply:
Pinging on this comment again. I don't think these constructors need to accept `maxIndent`, given `MaxIndent` is a constant that's just being passed down (when it was already accessible, just used incorrectly). It'll work as-is, it's just a little weird.